ui/src/common/ToggleFieldsMenu.js

```diff
@@ -82,6 +84,7 @@ export const ToggleFieldsMenu = ({
         }}
       >
         {TopBarComponent && <TopBarComponent />}
+        {albumView && <Slide />}
```

Comment:
This component should not know anything about the view being used. That's why we introduced the `TopBarComponent`. The Slide should be inside the `TopBarComponent` passed in the AlbumList